blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
Why would it need to be callable? Right now, the `callback` argument is a string. This class just adds an equality operator, but it still *is* the callback as we pass it to `do`, it's not the name of something else out there. If you call it `CallbackName` you would need to call the arguments `callback_name ` as well. But like I said, a callback in our case *is* a string, so adding "name" just confuses me.